app/bundles/CoreBundle/Helper/DateTimeHelper.php

```diff
@@ -43,13 +43,19 @@ class DateTimeHelper
      */
     private $datetime;
 
+    /**
+     * @var string
+     */
+    private $localTimezone;
+
     /**
      * @param \DateTimeInterface|string $string
      * @param string                    $fromFormat Format the string is in
      * @param string                    $timezone   Timezone the string is in
      */
     public function __construct($string = '', $fromFormat = 'Y-m-d H:i:s', $timezone = 'UTC')
     {
+        $this->localTimezone = ArrayHelper::getValue('default_timezone', (new ParamsLoaderHelper())->getParameters(), date_default_timezone_get());
```

Comment:
I think this is causing the tests to fail, because in https://github.com/mautic/mautic/pull/7752/files they're always using `date_default_timezone_get()`